src/orquestra/sdk/_base/abc.py

```diff
@@ -116,6 +116,29 @@ def get_available_outputs(
         """
         raise NotImplementedError()
 
+    @abstractmethod
+    def get_output(
+        self, workflow_run_id: WorkflowRunId, task_invocation_id: TaskInvocationId
+    ) -> WorkflowResult:
+        """Returns single output for a workflow run.
+
+        This method returns single artifact for given task.
+        When the task failed or was not yet completed, this method throws an exception
+
+        Careful: This method does NOT return status of a task.
+        Verify it beforehand to make sure if task failed/succeeded/is running.
+        You might get an exception
+
+        Args:
+            workflow_run_id: ID identifying the workflow run.
+            task_invocation_id: ID identifying single task run
+
+        Returns:
+            Whatever the task function returned, independent of the
+            ``@task(n_outputs=...)`` value.
```

Comment:
**suggestion**: Just for peace of mind, could you add an integration test that verifies that our Ray runtime implementation functions correctly with a task that has `n_outputs` > 1?

Reply:
By "function correctly" I mean it should be consistent with this docstring.
Given:
* a task def with `n_outputs=2`
* task returns a `"foo", "bar"`
Expect:
* `runtime.get_output(...)[0] == "foo"`
* `runtime.get_output(...)[1] == "bar"`

Reply:
rutnime.get_output[...] woudlnt work, as it returns JSON-encoded thing rather than tuple. But Ive added test to verify everythign works correctly